apis/concierge/config/v1alpha1/types_credentialissuer.go.tmpl

```diff
@@ -80,6 +80,22 @@ const (
 	ImpersonationProxyServiceTypeNone = ImpersonationProxyServiceType("None")
 )
 
+// ImpersonationProxyTLSSpec contains information about how the Concierge impersonation proxy should
+// serve TLS.
+type ImpersonationProxyTLSSpec struct {
+	// X.509 Certificate Authority (base64-encoded PEM bundle).
+	// Used to advertise the CA bundle for the impersonation proxy endpoint.
+	//
+	// +optional
+	CertificateAuthorityData string `json:"certificateAuthorityData,omitempty"`
+
+	// SecretName is the name of a Secret in the same namespace, of type `kubernetes.io/tls`, which contains
+	// the TLS serving certificate for the Concierge impersonation proxy endpoint.
```

Comment:
Let's document what will happen with the `ca.crt` data field of the Secret, and how that interacts with the `certificateAuthorityData` setting above, especially which takes precedent when both exist.

Reply:
Done, see next push.